internal/hud/renderer.go

```diff
@@ -239,38 +239,15 @@ func abbreviateLog(s string) []string {
 
 func (r *Renderer) renderResource(res view.Resource, rv view.ResourceViewState, selected bool) rty.Component {
 	layout := rty.NewConcatLayout(rty.DirVert)
+	renderResourceSummary(selected, rv, res, layout)
+	renderResourcesK8s(res, layout, rv)
+	renderResourceBuild(res, r, rv, layout)
```

Comment:
not a big deal either way, but I might prefer to write it like this:
```
layout.Add(renderResourceSummary(selected, rv, res))
layout.Add(renderResourcesK8s(res, rv))
layout.Add(renderResourceBuild(res, r, rv))
```

so that:
1. the methods are just constructing things rather than mutating
2. if you want to unit test one of these methods, you can just pass in the resource and resource view (or whatever) and check the resulting layout, without having to give it a layout